### ISSUE TITLE
Generate section "1.1 Namespace declarations" using -confFile option

### ISSUE THREAD

**Issue by arrascue** [dgarijo/Widoco#647]
Hello everyone,

I am not sure if this is a bug. I am using the latest version of Widoco (widoco-1.4.20-jar-with-dependencies_JDK-17.jar) to generate ontology documentation. 
I was initially using the option -getOntologyMetadata. 
With this option a section "1.1 Namespace declarations" was generated with the following table in it.

![image](https://github.com/dgarijo/Widoco/assets/22982088/b95d6a8f-f4a4-458e-9044-f7382b05bfa6)

When I switched to the option -confFile this section disappeared.
Moreover, it seems there is no meta-data key I could find in https://github.com/dgarijo/Widoco/blob/master/doc/metadataGuide/guide.md to add that section.

I was therefore wondering whether this could be a bug or I am missing something. 

For any help, I'd be very thankful.

Best regards,
Anthony Arrascue

**Comment by dgarijo** [comment]
@arrascue, that section is created automatically from the namespaces you use in your ontology. Not sure why it disappeared when you used the confFile option...

**Comment by Janakiram916** [comment]
Hello @dgarijo, I do experience the same issue regarding the section "1.1 Namespace declarations". 

1. In the first scenario, I generated the ontology document by using `-getOntologyMetadata` option. In this process, documentation contains namespace table. I also used `-saveConfig PATH` option to generate config.properties file for the present ontology configuration.
2. In the second scenario, I used this generated config file from the first scenario to generate the documentation. This time I used `-confFile PATH` option. Unfortunately, the generated document doesn't contain namespace table. 

<img width="756" alt="Screenshot 2023-11-27 at 13 42 01" src="https://github.com/dgarijo/Widoco/assets/56963950/f3f94d56-5abb-42a3-a1d5-068c4f76d9a6">
